fitbenchmarking/results_processing/support_page.py

```diff
@@ -79,9 +80,11 @@ def create_prob_group(prob_results, group_name, support_pages_dir,
         root = os.path.dirname(inspect.getfile(fitbenchmarking))
         template_dir = os.path.join(root, "templates")
         env = Environment(loader=FileSystemLoader(template_dir))
-        style_css = os.path.join(template_dir, 'main_style.css')
-        table_css = os.path.join(template_dir, 'table_style.css')
-        custom_style = os.path.join(template_dir, 'custom_style.css')
+        local_css_dir = os.path.join(options.results_dir,'css')
+        rel_css = os.path.relpath(local_css_dir,options.results_dir)
+        style_css = os.path.join(rel_css, 'main_style.css')
```

Comment:
It seems that when I run an example the style sheet are not correctly used in the support pages
![Screenshot from 2020-10-05 11-12-15](https://user-images.githubusercontent.com/15614849/95068103-b699f380-06fc-11eb-86ab-442a15db1e53.png)